monero/wallet_client.go

```diff
@@ -95,47 +136,22 @@ type walletClient struct {
 
 // NewWalletClient returns a WalletClient for a newly created monero-wallet-rpc process.
 func NewWalletClient(conf *WalletClientConf) (WalletClient, error) {
-	if conf.WalletFilePath == "" {
-		panic("WalletFilePath is a required conf field") // should have been caught before we were invoked
-	}
-
 	if path.Dir(conf.WalletFilePath) == "." {
 		return nil, errors.New("wallet file cannot be in the current working directory")
 	}
```

Comment:
Does this check belong in `Fill()` or not? It's kind of an arbitrary decision on what Fill is supposed to accomplish.  Whatever you decide is fine, as long as it was considered.

Reply:
`Fill` doesn't do anything with `WalletFilePath`, so I left it here